.github/workflows/ci.yml

```diff
@@ -242,17 +249,26 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-node-
       - name: Check libraries version
-        if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags/v2.')
+        if: |
+          (github.event_name == 'push' && startsWith(github.ref, 'refs/tags/v2.'))
+          ||
+          (github.event_name == 'push' && startsWith(github.ref, 'refs/tags/v3.'))
         run: ./scripts/ui_release.sh --check-package "$(echo ${{ github.ref_name }}|sed s/v2/v0/)"
```

Comment:
Should this `sed` be adjust as well?
(Not sure about what it does, if there is any collision risk making the change)

Reply:
hmm that is a good point and raises another: What are we going to do with go module versions :thinking: